tests/SearchHeadlessProvider.test.tsx

```diff
@@ -1,23 +1,19 @@
-import { SearchHeadlessProvider, SandboxEndpoints } from '../src';
+import { SearchHeadlessProvider } from '../src';
 import { render } from '@testing-library/react';
-import { provideHeadless } from '@yext/search-headless';
+import { provideHeadless, SearchHeadless } from '@yext/search-headless';
 
 jest.mock('@yext/search-headless', () => ({
-  provideHeadless: jest.fn(() => ({
-    setSessionTrackingEnabled: jest.fn(),
-    setSessionId: jest.fn()
-  }))
+  provideHeadless: jest.fn(() => { return {}; })
 }));
 
-it('correctly passes through a search config with sandbox endpoints', () => {
+it('does not change anything about the supplied Headless instance', () => {
```

Comment:
I feel like now just tests that no error is thrown? it doesn't really test that nothing is changed. I was thinking of just changing the name

Reply:
Because the mocked Headless instance has no methods, an error will be thrown if the `Provider` tries to invoke any method on the user-supplied Headless instance. That's the way I came up with to ensure the `Provider` doesn't much with the Headless instance in any way.

Reply:
ohh gotcha that makes sense. 
Maybe the test name could be changed to say that we don't call any headless methods within the provider? It's maybe a little literal, but I feel like it's not guaranteed that nothing changes about the headless instance. For instance we could wrap it (maybe to "decorate answers actions?") or add additional properties to it. 

Instead of returning an empty object, we could also return a Proxy that freaks out (throws an error) if it gets modified in any way

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy we could add get and set handlers to just throw an Error